data/org.freedesktop.UDisks2.xml

```diff
@@ -165,6 +165,9 @@
         Before the array is created, all devices in @blocks are
         erased. Once created (but before the method returns), the RAID
         array will be erased.
+
+        The @bitmap option can have the same values as are accepted by
+        SetBitmapLocation.
```

Comment:
Also, looking at the `mdadm` manpage, in case `bd_md_create(bitmap=False)`, libblockdev doesn't add any `--bitmap=` option, leaving the logic on automatic mdadm rule:

> When creating an array on devices which are 100G or larger, mdadm automatically adds an internal bitmap as it will usually be  beneficial.  This can be suppressed with --bitmap=none or by selecting a different consistency policy with --consistency-policy.

So either the values in UDisks should read _'auto'_ and _'internal'_, or we should change `bd_md_create()` to accept values of _'none'_, _'auto'_ and _'internal'_. Intentionally omitting _'clustered'_ and filename as those we don't want to support.

Reply:
> > When creating an array on devices which are 100G or larger, mdadm automatically adds an internal bitmap as it will usually be  beneficial.  This can be suppressed with --bitmap=none or by selecting a different consistency policy with --consistency-policy.

Oh, interesting, let's bring in the experts.  

@mauelsha, from our discussion in Brno I had the impression that `mdadm` will never use a bitmap but that it is always a good idea to use one.  Now the man page seems to say that a bitmap is not always a good idea, and that `mdadm` does in fact decides whether or not to use a bitmap exactly when it is a good idea.

Can you clarify?

Right now I am thinking that we don't need to change anything, after all.  New mdraid devices will have a bitmap or not as decided by the mdraid maintainers, and Cockpit allows it to be adjusted in case the maintainers are wrong in certain situations.